src/tentaclio/credentials/reader.py

```diff
@@ -85,6 +90,25 @@ def _load_from_file(
         raise TentaclioFileError("File not found") from e
 
 
+def replace_env_variables(url: str) -> str:
+    """Replace the environment variables inside the url by its value in the environment.
+
+    Environment variables match the following format: ${ENV_VARIABLE_NAME}
+    """
+    pattern = re.compile(ENV_VARIABLE_PATTERN)
+    env_variables = re.findall(pattern, url)
+    for env_variable in env_variables:
+        env_variable_name = re.sub(NOT_VALID_ENV_VARIABLE_VALUES, "", env_variable)
+        env_value = os.getenv(env_variable_name)
+        if not env_value:
+            raise EnvironmentError(
+                f"Error while reading variable '{env_variable_name}' from the environment. "
```

Comment:
Can you add that this happened when interpolating the variable from the the secret file?